src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py

```diff
@@ -1757,5 +1757,131 @@ def test_private_endpoint_connection_search(
+    self.kwargs.update({
+        'resource': self.create_random_name('cli-test-resource-', 24),
+        'vnet': self.create_random_name('cli-vnet-', 24),
+        'subnet': self.create_random_name('cli-subnet-', 24),
+        'pe': self.create_random_name('cli-pe-', 24),
+        'pe_connection': self.create_random_name('cli-pec-', 24),
+    })
+
+    # create resource
+    self.kwargs['extra_create'] = self.kwargs.get('extra_create', '')
+    self.kwargs['show_name'] = self.kwargs.get('show_name', '-n')
+    self.kwargs['create_name'] = self.kwargs.get('create_name', '-n')
+    self.cmd('{cmd} create -g {rg} {create_name} {resource} {extra_create}')
+    result = self.cmd('{cmd} show -g {rg} {show_name} {resource}').get_output_in_json()
+    self.kwargs['id'] = result['id']
+
+    # test private-link-resource
+    result = self.cmd('network private-link-resource list --name {resource} -g {rg} --type {type}',
+                      checks=self.check('length(@)', '{list_num}')).get_output_in_json()
+    self.kwargs['group_id'] = result[0]['properties']['groupId'] if group_id else result[0]['groupId']
+
+    # create private-endpoint
+    self.cmd('network vnet create -n {vnet} -g {rg} --subnet-name {subnet}')
+    self.cmd('network vnet subnet update -n {subnet} --vnet-name {vnet} -g {rg} '
+             '--disable-private-endpoint-network-policies true')
+
+    self.cmd('network private-endpoint create -g {rg} -n {pe} --vnet-name {vnet} --subnet {subnet} '
+             '--connection-name {pe_connection}  --private-connection-resource-id {id} --group-id {group_id}',
+             checks=self.check('privateLinkServiceConnections[0].privateLinkServiceConnectionState.status', 'Approved'))
+
+    # test private-endpoint-connection
+    result = self.cmd('network private-endpoint-connection list --name {resource} -g {rg} --type {type}',
+                      checks=self.check('length(@)', 1)).get_output_in_json()
+    self.kwargs['name'] = result[0]['name'] if list_name else result[0]['id'].split('/')[-1]
+    # For some services: A state change from Approved to Approved is not valid
+    if approve:
+        self.cmd('network private-endpoint-connection approve --name {name} -g {rg} '
+                 '--resource-name {resource} --type {type}')
+
+    self.cmd('network private-endpoint-connection show --name {name} -g {rg} --resource-name {resource} --type {type}',
+             checks=self.check('properties.privateLinkServiceConnectionState.status', 'Approved'))
+
+    self.cmd('network private-endpoint-connection reject --name {name} -g {rg} '
+             '--resource-name {resource} --type {type}',
+             checks=self.check('properties.privateLinkServiceConnectionState.status', 'Rejected'))
+
+    self.cmd('network private-endpoint-connection delete --name {name} -g {rg} '
+             '--resource-name {resource} --type {type} -y')
+
+
+class NetworkPrivateLinkScenarioTest(ScenarioTest):
+    @live_only()
```

Comment:
What's the reason you use `live_only` here?

Reply:
These test rely on cmd of other modules. To avoid being broken by other module frequently, we only run live test.

Reply:
Removed `live_only`

Reply:
done